src/MySqlConnector/Serialization/ErrorPayload.cs

```diff
@@ -21,7 +21,7 @@ public static ErrorPayload Create(PayloadData payload)
 			reader.ReadByte(Signature);
 
 			var errorCode = reader.ReadUInt16();
-			reader.ReadByte(0x23);
+			reader.ReadByte(0x23, 0x48);
```

Comment:
`0x23` is `#`, so I guess I interpreted "# marker of the SQL State" as meaning it was the constant `'#'`.

Still not sure what `0x48` (== `'H'`) means, but it must be an acceptable state marker. Since we're not doing anything with that value, it seems safe to ignore it if the rest of the packet looks right.